app.py

```diff
@@ -53,12 +55,18 @@ def _create_email_subject() -> str:
 
 def send_email() -> None:
     """Send e-mail with begave test reports."""
+    msg = MIMEMultipart()
     with open(_BEHAVE_REPORT_FILE, "r") as fp:
-        msg = MIMEText(fp.read(), "html")
+        report = fp.read()
 
     msg["Subject"] = _create_email_subject()
     msg["From"] = _EMAIL_FROM
     msg["To"] = _EMAIL_TO
+    msg.attach(MIMEText(report, "html"))  # Message body.
```

Comment:
report might be used uninitialized here?!

Reply:
It is read on line 60.

Reply:
ja, I saw that. How does it behave if _BEHAVE_REPORT_FILE is not present?

Reply:
The HTML report is generated only if an e-mail is requested to be sent:

https://github.com/thoth-station/integration-tests/blob/3e6cd68c48023d24858e46fed4caebaa7c5881bf/app.py#L71-L72

Otherwise logs are printed to stdout (the default behavior without `["-f", "html", "-o", "behave-report.html"]`) arguments.